src/controllers/P_TemplateController.php

```diff
@@ -97,7 +97,7 @@ public function do_add(array $argv = [], array $request = []) {
         }
 
         $this->View->Node       = $Node;
-        $this->View->formAction = '/P_Template/add';
+        $this->View->formAction = '/P_Template/add'.$Node->node_id;
```

Comment:
The node_id doesn't exist yet, and this could break the form action.
```suggestion
        $this->View->formAction = '/P_Template/add';
```